ledger/sandbox/src/main/scala/com/digitalasset/platform/store/dao/JdbcLedgerDao.scala

```diff
@@ -946,12 +948,29 @@ private class JdbcLedgerDao(
 
     val txBytes = serializeTransaction(ledgerEntry.entry)
 
+    def splitOrThrow(id: EventId): NodeId =
+      split(id).fold(sys.error(s"Illegal format for event identifier $id"))(_.nodeId)
+
     def insertEntry(le: PersistenceEntry)(implicit conn: Connection): PersistenceResponse =
       le match {
         case PersistenceEntry.Transaction(tx, globalDivulgence, divulgedContracts) =>
           Try {
             storeTransaction(offset, tx, txBytes)
 
+            // TODO Run this directly from the indexer once we no longer
+            // TODO need to validate transactions on the index
```

Comment:
Can we make an issue for this rather than leaving long-lived comments in the code?

Reply:
Sure, make sense, thanks.

Reply:
Addressed by https://github.com/digital-asset/daml/pull/5029/commits/69c89d1cd151749414508f0e893d3f606beda166 and #5035.

Reply:
Thanks!